### DISCUSSION TITLE
Would love some, Voxglitch Autobreak Studio

### DISCUSSION THREAD

**Discussion by 333calvin333** [DISTRHO/Cardinal#677] [General]
With all due respect and thank you from the bottom of my heart for Cardinal, feels bad to ask but would it be possible to add the following moduel? Voxglitch Autobreak Studio. As cANARd (love it but it) can be a little cramped UI wise at times. I'm sure in future someone dabbling in one of the breakbeats driven genres may appreciate. Kindest regards and again thankyou for Cardinal.

**Comment by dromer** [comment]
Voxglitch plugin first needs updating, which is being tracked here: https://github.com/DISTRHO/Cardinal/issues/638
After that we can enable this specific module as well.

**Comment by cosinekitty** [comment]
Let's ping the Voxglitch developer himself... hey @clone45, we're talking about you, LOL!

**Comment by clone45** [comment]
Hi everyone!  Thanks for the heads up, Don.  I was just talking to the cardinal team a few days ago about updates.  The situation is that my modules need some work to be more compatible with Cardinal before we can get them updated.  Specifically, the fancy front panels that I've created put a strain on Cardinal working on the web, and it's likely that I'll need to simplify the front panels in order to make my collection more compatible with non-VCV Rack platforms.  I do hope to return to module development, but I can't for a little while.  I sincerely apologize for the delay, but it could be a while.  I'd like to thank the Cardinal team for being outstanding and easy to work with.  :-)
